.chloggen/dataset-exporter-make-shutdown-timeout-configurable.yaml

```
@@ -0,0 +1,20 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: datasetexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Make duration of shutdown procedure configurable to minimise data losses."
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [24415]
```

```suggestion
issues: [24252]
```
Since there's an issue open for this PR, we should use that instead of the PR number itself.

---

This second changelog does not have any issue. It's just implemented in this PR. And the comment says that it should be either PR or issue. Therefore I have specified this PR.